packages/fxa-settings/src/components/Modal/index.tsx

```diff
@@ -28,6 +31,9 @@ export const Modal = ({
   hasButtons = true,
   headerId,
   descId,
+  widthClassName = 'max-w-lg',
```

Comment:
It looks like all modals are the same width in the Figma doc, is there are specific reason for needing varying modal max-widths? It might be best to make them consistent. Looks like `max-w-md` is closest, but if we do that we should reduce the horizontal padding so "Disable two-step authentication?" doesn't break to two lines